core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -364,6 +369,16 @@ private[spark] class TaskSchedulerImpl(
     launchedTask
   }
 
+  /**
+   * Check whether the resources from the WorkerOffer are enough to run at least one task.
+   */
+  private def resourceMeetTaskRequirements(
```

Comment:
name "resourcesMeetTaskRequirements"  since we could have multiple

Reply:
updated